audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/stream/config/AkkaComponentsConfig.java

```diff
@@ -73,6 +74,7 @@ public AkkaComponentsConfig(final AuditServiceConfigProperties configProperties)
         this.configProperties = configProperties;
     }
 
+    @SuppressWarnings("resource")
```

Comment:
Are all these due to Eclipse throwing warnings?

Reply:
Yeah. I have quite strict compiler level warnings turned on. Eclipse then throws up quite a few issues. Also a good clue to the developer that there is potentially a resource issue.